Sources/Orbit/Components/ListChoiceGroup.swift

```diff
@@ -11,28 +20,79 @@ public struct ListChoiceGroup<Content: View>: View {
 
     @Environment(\.horizontalSizeClass) var horizontalSizeClass
 
-    let status: Status?
-    let backgroundColor: Color?
+    let title: String
+    let iconContent: Icon.Content
+    let style: ListChoiceGroupStyle
+    let titleStyle: Header.TitleStyle
     let content: () -> Content
 
     public var body: some View {
-        Card(spacing: 0, padding: 0, style: .iOS, status: status, backgroundColor: backgroundColor) {
-            content()
+        switch style {
+            case .card(let status, let backgroundColor):
+                Card(title, spacing: 0, padding: 0, style: .iOS, status: status, backgroundColor: backgroundColor) {
+                    content()
+                }
+            case .borderless:
+                VStack(alignment: .leading, spacing: .medium) {
+                    header
+
+                    VStack(alignment: .leading, spacing: 0) {
+                        content()
+                    }
+                }
+                .background(Color.white)
```

Comment:
Borderless variant should be with no background.

Reply:
That makes previews look funny, since `ListChoice` views have a default white background. What do we do about that?